#163445309 data structure changed to database

samorahno opened the pull request:
#### What does this PR do?
This PR implements changing the data structure used on this app to database
#### Description of Task to be completed?
- Data structure for challenge 2 changed to database
#### How should this be manually tested?
By cloning the [Gith Repo](https://samorahno.github.io/Politico/UI) 
use postman to test the endpoints.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
#163445309
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/45196374/52000466-757c2580-24bd-11e9-923d-086b410fba14.png)

#### Questions:
None

samorahno committed:
feature (database):datastructure changed to database

- data structure for challenge 2 changed to database

Delivers #163445309

andela-gike commented on server/controllers/PoliticalParty.js:
```diff
@@ -1,103 +1,129 @@
+        });
+      }
+      return res.status(202).json({
+        status: 202,
```
Kindly confirm if this is the right status code

samorahno replied:
Yes i confirmed, it is suppose to be 200 though, but since i used async await, it shows that the delete is done asynchronously. 
my reference - https://restfulapi.net/http-status-codes/
![image](https://user-images.githubusercontent.com/45196374/52044334-3e038c80-2542-11e9-8025-866e0c67044c.png)

andela-gike replied:
But your response message says delete was successful, which means the async request was carried out. `A 202 response is typically used for actions that take a long while to process. It indicates that the request has been accepted for processing, but the processing has not been completed.`

samorahno committed:
feature (database):datastructure changed to database

- data structure for challenge 2 changed to database update

Delivers #163445309
